packages/tooling/fast-figma-plugin-msft/manifest.json

```diff
@@ -1,7 +1,10 @@
 {
-  "name": "Theme Designer (Alpha)",
+  "name": "Fluent UI Design Tokens (Alpha)",
```

Comment:
Should this be "FAST" and not "Fluent?

Reply:
I'm taking some liberties that the package is still "-msft". I've bundled the Fluent components and titled it as such for internal testing first. For further testing I think it needs what we discussed regarding Creator being able to import or at least toggle on the different component libraries.